internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -1146,10 +1120,12 @@ private ConfigKeys() {
       "native.replication.source.fabric.as.default.for.hybrid.stores";
 
   /**
-   * The default source fabric used for native replication for incremental push stores.
+   * We will use this config to determine whether we should enable incremental push for hybrid active-active user stores.
+   * If this config is set to true, we will enable incremental push for hybrid active-active user stores.
    */
-  public static final String NATIVE_REPLICATION_SOURCE_FABRIC_AS_DEFAULT_FOR_INCREMENTAL_PUSH_STORES =
-      "native.replication.source.fabric.as.default.for.incremental.push.stores";
+  public static final String ENABLE_INCREMENTAL_PUSH_FOR_HYBRID_ACTIVE_ACTIVE_USER_STORES =
```

Comment:
nit: not big deal, but do you want to align with the "as_default" style for this new config?